lib/FrontendTool/ReferenceDependencies.cpp

```diff
@@ -129,25 +129,23 @@ bool swift::emitReferenceDependencies(DiagnosticEngine &diags,
                                       const FrontendOptions &opts) {
```

Comment:
In the formulation where the ReferencedNameTracker knows its path, `opts` is never used. You can decide what you want to do with that, though.

Reply:
That gives me an idea! Maybe the path lookup can be done right there. (I know you were hinting at that, tx!)